ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -157,22 +156,6 @@
         option: external_enabled
         value: 'true'
 
-    - name: Grafana                    | Set snapshot server
-      when: not grafana_db.stat.exists
-      ini_file:
-        dest: /etc/grafana/grafana.ini
-        section: snapshots
-        option: external_snapshot_url
-        value: https://snapshots-g635.percona.com/graph/
-
-    - name: Grafana                    | Set snapshot server
-      when: not grafana_db.stat.exists
-      ini_file:
-        dest: /etc/grafana/grafana.ini
-        section: snapshots
-        option: external_snapshot_name
-        value: Share with Percona
-
```

Comment:
Removing, since these are duplicates of https://github.com/percona/pmm-update/blob/15e211a9dba7ec3bd02124366602e3e12a55a96f/ansible/playbook/tasks/update.yml#L140-L152

Reply:
Do we run pmm-update ansible scripts on each build? Where do we do it?